### ISSUE TITLE
rose bush: handle accidental HTML tags in output

### ISSUE THREAD

**Issue by benfitzpatrick** [metomi/rose#1684]
A line in job output with `<` or `>` tag-like strings, such as:

```
Oh no! n_coffees <min_coffees
```

will be interpreted as a bad HTML tag and will break the display of subsequent lines in the output.

We could try to detect the accidental HTML and escape it, or we could drop all support for it in
job output. A middle ground would be to escape all `>`, `<`, and `&` by default, except for things that match known tags like `a`, `h1`, etc.

**Comment by matthewrmshin** [comment]
We do support a raw view for this purpose.

**Comment by matthewrmshin** [comment]
And the text view as well.

**Comment by benfitzpatrick** [comment]
Maybe those should be the default in these situations.

**Comment by nhsavage** [comment]
Possibly this is just about promoting greater user awareness of these issues. When I came across this problem

a. I didn't notice the "text" view 
b. even if I had, I wouldn't have necessarily thought to use it

It was only when Stuart W suggested that I look at the raw output that I realised it might be an issue with displaying (as opposed to fetching it). Then I didn't use the raw or text view (both of which are fine now I look at them) but instead opened the file from the command line.
